### DISCUSSION TITLE
Error decrypting 200MB of attached files

### DISCUSSION THREAD

**Discussion by renussunre** [PrivateBin/PrivateBin#1153] [Q&A]
### Describe the problem/question

Hi, I have a PrivateBin 1.5.2 with Apache2

I've changed the upload_max_filesize, post_max_size and max_execution_time, in the php.ini of apache2. 
And i've tried other things, like using msyql. 
The fact is that I wanted to share files with my own server without using Cloud's. But I only can post/get files of 100Mb. 
When I try with 200mb fails to download. The web gets stuck with ''Decryption paste''
<img width="251" alt="image" src="https://github.com/PrivateBin/PrivateBin/assets/142978303/9dfdfb9a-2d69-40b8-84ec-032a9a814369">

And if I look into the log, I only see that ''works fine''.
<img width="661" alt="image" src="https://github.com/PrivateBin/PrivateBin/assets/142978303/6566b7c0-cd76-4b85-b159-08149b23737c">

¿Wich limit size have you achieve with these project? ¿Is there a way for doing that? I know that the project is only for lower size, but I wanted to implement for a private share files.

### Did you use the FAQ section?

- [X] Yes, I have read the FAQ and I found no solution/answer there.

### What you did?

1.
2.
...

### PrivateBin version

1.5.2

### Issue reproducibility

Yes

**Comment by rugk** [comment]
"Decrypting paste'" means the paste has been downloaded successful and you actually have an error while well… decrypting it in your client i.e. browser. As such, it is to be expected that there are no web server errors and the problem lies in the client.

Now encrypting and decrypting 200MB of data is a lot for a browser. I am actually surprised 100MB went well for you, because see https://github.com/PrivateBin/PrivateBin/issues/858 for other issues that can appear with that large files.

AFAIK also the encryption is no stream-encryption although it is WebEncryption, which is at least natively implemented in the browser.
I am sorry, but I doubt you can use the project for such big files. I would personally recommend https://wormhole.app/ for sharing large (end-to-end encrypted) files, which also has p2p support, so files do not even need to get uploaded, but that is off-topic as for this project.
